examples/data-objects/clicker/src/agent.ts

```diff
@@ -3,11 +3,11 @@
  * Licensed under the MIT License.
  */
 
-import { IFluidRouter, IFluidRunnable, IRequest, IResponse } from "@fluidframework/core-interfaces";
+import { IFluidRunnable, IRequest, IResponse } from "@fluidframework/core-interfaces";
 import { SharedCounter } from "@fluidframework/counter";
 
 // Sample agent to run.
-export class ClickerAgent implements IFluidRouter, IFluidRunnable {
+export class ClickerAgent implements IFluidRunnable {
```

Comment:
Mark request and IFluidRouter as deprecated below too?  I'm curious if there's any running scenario where this object's IFluidRouter getter is actually invoked.  Not sure what kind of thing this is.

Reply:
And likewise in other classes in the data-objects dir.

I guess could come as a follow-up, don't want to block this PR necessarily on these examples.

Reply:
For this specific usage (agent.ts) should be safe to just remove.